.gitignore

```diff
@@ -133,3 +133,10 @@ msvc-full-features/distribution
 
 # Transifex Client
 .tx/
+
+# Generic Files
+.cproject
+.settings/
+data/mods/nocs_cata_mod/
+data/sound/ChestHole/
```

Comment:
`git checkout -- <file>` to make it not complain, or ([better](https://help.github.com/articles/ignoring-files/)) you can add these to /path/to/cdda/.git/info/exclude. We don't want to change everyone's .gitignore =)